assets/queries/cloudFormation/sns_topic_subscription_not_setted/test/positive.yaml

```diff
@@ -0,0 +1,7 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: ''
+Resources:
+  SnsTopic:
+    Type: 'AWS::SNS::Topic'
+    Properties:
+      TopicName: alarm-action
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline